ssg/build_remediations.py

```diff
@@ -656,17 +656,20 @@ def get_rule_dir_remediations(dir_path, remediation_type, product=None):
         return []
 
     results = []
+    common_results = []
     for remediation_file in sorted(os.listdir(remediations_dir)):
         file_name, file_ext = os.path.splitext(remediation_file)
         remediation_path = os.path.join(remediations_dir, remediation_file)
 
         if file_ext == ext and rules.applies_to_product(file_name, product):
             if file_name == 'shared':
-                results.append(remediation_path)
+                common_results.append(remediation_path)
+            elif file_name != product:
```

Comment:
Nitpick: I think it is a bit confusing that it checks that `file_name != product`.
As `file_name` passed through `applies_to_product()` it is clearly a sub-string of `product`, i.e. "ubuntu", when product is "ubuntu2004".

I'd add a comment here, or flip the order, `elif file_name == product:` and the `else` handles any product sub
 -string

Reply:
Sure. :) Yeah, I grouped it this way since grouping `common_results` modifications ahead of `results` modifications made most sense to me but I'll clarify with a comment.

Reply:
You kinda overdid it,but looks ok, :)